process_data.sh

```diff
@@ -143,11 +143,22 @@ file_t2_seg=$FILESEG
 # Flatten scan along R-L direction (to make nice figures) 
 sct_flatten_sagittal -i ${file_t2}.nii.gz -s ${file_t2_seg}.nii.gz
 
-# Bring vertebral level into T2 space 
-sct_register_multimodal -i label_T1w/template/PAM50_levels.nii.gz -d ${file_t2_seg}.nii.gz -o PAM50_levels2${file_t2}.nii.gz -identity 1 -x nn
+# Dilate t2 cord segmentation to use as mask for registration
+file_t2_mask="${file_t2_seg}_dil"
+ImageMath 3 ${file_t2_mask}.nii.gz MD ${file_t2_seg}.nii.gz 40
```

Comment:
the input to `ImageMath` is `${file_t2_mask}`, but the variable has just been updated with suffix "_dil" one line above, so the file does not exist yet (or am I missing something?)

Reply:
or maybe the input/output are swapped?

Reply:
The usage for `ImageMath` is:
~~~
ImageMath ImageDimension <OutputImage.ext> [operations and inputs] <Image1.ext>
~~~
The output file is before the input file in the syntax, that's why!

Reply:
wow! that's a bit unusual 😅